src/TypedData/DataFetcher.php

```diff
@@ -98,6 +99,7 @@ public function fetchDataBySubPaths(TypedDataInterface $typed_data, array $sub_p
       $current_selector = implode('.', $current_selector);
       throw new \InvalidArgumentException("Unable to apply data selector '$selector' at '$current_selector': " . $e->getMessage());
```

Comment:
here is another global \InvalidArgumentException that you forgot? Make sure to grep the code base for all remaining global exceptions